examples/flutter_login/lib/login/view/login_form.dart

```diff
@@ -80,13 +80,13 @@ class _LoginButton extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return BlocBuilder<LoginBloc, LoginState>(
-      buildWhen: (previous, current) => previous.status != current.status,
+      buildWhen: (_, current) => current.isValid,
```

Comment:
I don't think this is a correct way to define it. Login button won't rebuild if login form not valid, therefore button cannot disable after enabled for example by erase all input. And it's rebuild when isValid, even though type single character will rebuild the button.

Here is my suggestion:
```dart
buildWhen: (previous, current) =>
  previous.status != current.status ||
  previous.isPure != current.isPure ||
  previous.isValid != current.isValid,
```